### PR TITLE
Feature/sof 4118

### PR DESCRIPTION
**Pull request by adewyer**
Update of nwchem output file to match RuPy testing.

### CODE REVIEW

File: tests/fixtures/nwchem/test-001/run.sh [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 module add nwchem/702-g-485-ompi-110
 
-mpirun --allow-run-as-root -np 1 nwchem.inp &> nwchem.log
+mpirun --allow-run-as-root -np 1 nwchem.inp &> nwchem-sp-energy.log
```
Comment (timurbazhirov) [inline tests/fixtures/nwchem/test-001/run.sh:6]: What does "sp" stand for?

Comment (adewyer) [reply tests/fixtures/nwchem/test-001/run.sh:6]: single point energy, so it is a snapshot of the energy at that specific geometry, there is no optimization of the structure involved.

Comment (adewyer) [reply tests/fixtures/nwchem/test-001/run.sh:6]: Maybe calling it just "nwchem-energy.log" or "nwchem-total-energy.log" would be better?

Comment (triplepoint) [reply tests/fixtures/nwchem/test-001/run.sh:6]: Either that or a short comment here as a reminder would be fine.

**Comment by adewyer** [comment]
Currently pymatgen error with python 3.7 is causing the build to fail. Python 3.6 build was successful.

File: tests/fixtures/nwchem/test-001/run.sh (hunk 1 above)
Comment (adewyer) [reply tests/fixtures/nwchem/test-001/run.sh]: Changed the naming

**Comment by triplepoint** [comment]
I see this is PR is set to be merged back to the epic branch `epic/SOF-4116-nwchem` but the other pull requests on the nwchem work all are merging back to their respective `dev` branches, including #64 on this repo.  Should this also merge to `dev`?

**Review by triplepoint — APPROVED** [review]
sheep_lgtm

**Pull request merged**